templates/client.ts.hbs

```diff
@@ -93,6 +93,14 @@ export class {{name.upperCamelCase}}<ConfigParameters extends {{{name.upperCamel
 
   static readonly defaultBaseUri = "{{getBaseUriFromDocument model}}";
 
+  static readonly apiPaths = {
```

Comment:
Creates a read  only `apiPaths` object that maps the API endpoint name to the API path